src/pages/docs/argo-cd/annotations/helm-annotations.md

```diff
@@ -227,73 +231,33 @@ different:
       image: busybox:1
 ```
 
+
 #### Example 4
 
-A Helm source that references multiple values files from the source repo.
+A Helm source that has multiple ref sourced values files. Each ref source can be updated by a different project if required.
 
 ```yaml
 ...
 metadata:
   annotations:
-    argo.octopus.com/project: "proj-1"
-    argo.octopus.com/environment: "development"
-
-    # When there are multiple sources, we need an annotation to tell us which file path annotations belong to (Note: the actual name of the alias can be arbitrary)
-    # if an alias is not provided for a path, that values file will be ignored
-    argo.octopus.com/image-replace-alias.core: "app-files/values.yaml"
-    argo.octopus.com/image-replace-alias.overlay: "app-files/values-overlay.yaml"
+    argo.octopus.com/project.remote-source: "proj-1"
+    argo.octopus.com/environment.remote-source: "development"
+    argo.octopus.com/project.other-source: "proj-2"
+    argo.octopus.com/environment.other-source: "development"
 
-    argo.octopus.com/image-replace-paths.core: "{{ .Values.image.name}}:{{ .Values.image.version}}"
-    argo.octopus.com/image-replace-paths.overlay: "{{ .Values.different.structure.here.image }}"
-...
-spec:
-  sources:    
-    - repoURL: https://github.com/my-org/my-argo-helm-app
-      path: "chart"
-      targetRevision: main
-      helm:
-        valueFiles:
-          - app-files/values.yaml
-          - app-files/values-overlay.yaml
-```
-
-**app-files/values.yaml**
-```yaml
-image:
-  name: nginx/nginx
-  version: 1.19.0
-```
-**$remote-values/values-overlay.yaml**
-```yaml
-different:
-  structure:
-    here:
-      image: busybox:1
-```
-
-#### Example 5
-
-A Helm source that has multiple ref sourced values files.
-
-```yaml
-...
-metadata:
-  annotations:
-    argo.octopus.com/project: "proj-1"
-    argo.octopus.com/environment: "development"
-
-    argo.octopus.com/image-replace-paths.other-values: "{{ .Values.another-image.name }}"
-    argo.octopus.com/image-replace-paths.remote-values: "{{ .Values.image.name}}:{{ .Values.image.version}}"
+    argo.octopus.com/image-replace-paths.helm-source: "{{ .Values.another-image.name }}"
```

Comment:
same problem as previous - the two values files have different content which is not supported.